catalog/build_ci/B008.md

```diff
@@ -0,0 +1,8 @@
+### B008
+
+|**Code**           | **B008** |
+|**Title**          | CI/CD runner servers should be set up automatically.|
```

Comment:
```suggestion
|**Title**          | CI/CD agents installation and config should be in version control.|
```
I think it's title and description should be more abstract.
They might use server or whatever.
They might not setup the agent automatically, but the instructions or dependencies are specified clearly.
Consider a scenario that a team works with Github actions or they got their agents as service from another team